easybuild/easyblocks/o/openblas.py

```diff
@@ -55,7 +55,11 @@ def build_step(self):
             del os.environ[cflags]
             self.log.info("Environment variable %s unset and passed through command line" % cflags)
 
-        cmd = "%s make %s %s" % (self.cfg['prebuildopts'], ' '.join(build_parts), self.cfg['buildopts'])
+        makecmd = 'make'
+        if self.cfg['parallel']:
+            makecmd += "-j %s" % self.cfg['parallel']
```

Comment:
Ther is one space missing
```suggestion
            makecmd += " -j %s" % self.cfg['parallel']
```

Reply:
I did notice that, but forgot to squeeze in the space correctly before pushing... 🤦‍♂ 

Thanks for checking this carefully!